plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchTaskHandler.groovy

```diff
@@ -483,6 +484,17 @@ class AwsBatchTaskHandler extends TaskHandler implements BatchHandler<String,Job
         if( jobRole )
             container.setJobRoleArn(jobRole)
 
+        final logsGroup = opts.getLogsGroup()
+        if( logsGroup )
+            container.setLogConfiguration(
+                new LogConfiguration()
+                    .withLogDriver('awslogs')
+                    .withOptions([
+                        'awslogs-region': opts.getRegion(),
+                        'awslogs-group': logsGroup
+                    ])
+            )
```

Comment:
Since this is invoked for each task, let's move into a memorized method, eg. 


```
@Memoized 
LogConfiguration logsConfig(String name, String region) {
      new LogConfiguration()
                    .withLogDriver('awslogs')
                    .withOptions([
                        'awslogs-region': region,
                        'awslogs-group': name
                    ])
}
```

This saves (a bit) some classes creation and relative garbage to be collected